src/AllocatorOracle.sol

```diff
@@ -17,17 +17,15 @@
 pragma solidity ^0.8.16;
 
 contract AllocatorOracle {
```

Comment:
btw the reason we are not using ds-value here is to save the storage read?

Reply:
Not really, in the beginning because we needed a less standard value and using a fixed oracle would help to make sure to not mess with it. I'd guess today just because it is already done and it is easier for deploying having the contract there.